promql/engine_test.go

```diff
@@ -3150,452 +3148,6 @@ func TestInstantQueryWithRangeVectorSelector(t *testing.T) {
 	}
 }
 
-func TestNativeHistogram_Sum_Count_Add_AvgOperator(t *testing.T) {
```

Comment:
I first was surprised why this got removed here, while no framework-based test is added in lieu of it. But then I saw that those framework tests were added earlier. I'll update the commit description to include that fact to avoid future surprises.

Reply:
I was also surprised by this, as in #14585 it was removed but I couldn't find where it was added again.

Reply:
That easily happens when resolving merge conflicts.